app/components/baseComponents/RFTextInput/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { TextInput, View } from 'react-native';
+import { black } from '@scss/colors';
+
+import { styles } from './styles';
+
+class RFTextInput extends PureComponent {
+  onChangeText = () => true;
```

Comment:
why this?

Reply:
you are not using it

Reply:
I think i anticipated component functionality :laughing: